atom/browser/api/atom_api_web_contents.cc

```diff
@@ -1441,20 +1446,27 @@ void WebContents::Print(mate::Arguments* args) {
   print_view_manager_basic_ptr->PrintNow(
       web_contents()->GetMainFrame(), settings.silent,
       settings.print_background, settings.device_name);
+#endif
```

Comment:
perhaps we should `throw` or otherwise signal the fact that printing is disabled?

Reply:
Do we throw errors for other disabled features too? AFAIK OSR for example just silently does nothing when disabled, but there might be other feature that throw, not sure. Good idea, I just wanted to be consistent with what I saw before.

Reply:
Makes sense to be consistent, I suppose. The use case I'm imagining is someone trying a nightly build and printing silently failing because we've turned it off temporarily, and being confused. If we at least `LOG`ged then they'd have a hope of figuring out what was going on :)